src/test/java/com/conveyal/datatools/manager/jobs/GisExportJobTest.java

```diff
@@ -0,0 +1,230 @@
+                // GeometryAttribute sourceGeometry = feature.getDefaultGeometryProperty();
+                Collection<Property> properties = feature.getProperties();
+                // Iterate over feature properties and verify everything looks OK.
+                for (Property property : properties) {
+                    String name = property.getName().toString();
+                    LOG.info("{}: {}", name, property.getValue());
+                    if ("the_geom".equals(name)) {
+                        // Check that the geometry was exported properly.
+                        assertThat(property.getValue(), notNullValue());
+                    }
+                }
+            }
+        }
+        // Ensure that all stops from feed version are present in shapefile.
+        assertThat(featureCount, equalTo(calTrainVersion.feedLoadResult.stops.rowCount));
+    }
+
+    @Test
+    public void canExportRoutes () throws IOException, SQLException {
+        // Run the GIS export job for stops.
+        File zipFile = File.createTempFile("routes", ".zip");
+        Set<String> ids = new HashSet<>();
+        ids.add(calTrainVersion.id);
+        GisExportJob gisExportJob = new GisExportJob(GisExportJob.ExportType.ROUTES, zipFile, ids, "test");
+        gisExportJob.run();
+        assertThat(gisExportJob.status.error, equalTo(false));
+        FeatureCollection collection = getFeatureCollectionFromZippedShapefile(zipFile);
+        // Iterate over features.
+        int featureCount = 0;
+        try (FeatureIterator iterator = collection.features()) {
+            while (iterator.hasNext()) {
+                featureCount++;
+                Feature feature = iterator.next();
+                // GeometryAttribute sourceGeometry = feature.getDefaultGeometryProperty();
+                Collection<Property> properties = feature.getProperties();
+                // Iterate over feature properties and verify everything looks OK.
+                for (Property property : properties) {
+                    String name = property.getName().toString();
+                    LOG.info("{}: {}", name, property.getValue());
+                    if ("the_geom".equals(name)) {
+                        // Check that the geometry was exported properly.
+                        assertThat(property.getValue(), notNullValue());
+                    }
+                }
+            }
+        }
+        PreparedStatement preparedStatement = DataManager.GTFS_DATA_SOURCE.getConnection()
+            .prepareStatement(
+                String.format("select count(*) from %s" + ".patterns", calTrainVersion.namespace));
+        ResultSet resultSet = preparedStatement.executeQuery();
+        int patternCount = 0;
+        while (resultSet.next()) {
+            patternCount = resultSet.getInt(1);
+        }
+        // Check that feature count = pattern count from SQL query.
+        assertThat(featureCount, equalTo(patternCount));
```

Comment:
Another place where it'd be good to have a basic check of the coordinates that were exported.